libs/openchallenges/ui/src/lib/organization-card/organization-card.component.ts

```diff
@@ -9,16 +10,60 @@ import { Avatar } from '../avatar/avatar';
 })
 export class OrganizationCardComponent implements OnInit {
   @Input() organization!: Organization;
+  @Input() personAvatarSize = 36;
```

Comment:
The input seems not been used.

Reply:
I'll go ahead and comment out this line as well - thanks!

(I had commented out the `mockMembers` data since we do not yet want to show the list of organization members)